docs/standard/threading/overview-of-synchronization-primitives.md

```diff
@@ -1,174 +1,167 @@
+
+You create a <xref:System.Threading.EventWaitHandle> instance that represents a named system synchronization event by using one of the [EventWaitHandle constructors](<xref:System.Threading.EventWaitHandle.%23ctor%2A>) that specifies a name or the <xref:System.Threading.EventWaitHandle.OpenExisting%2A?displayProperty=nameWithType> method. A named system synchronization event can be used for the inter-process synchronization.
+
+For more information, see the [EventWaitHandle](eventwaithandle.md) article and the <xref:System.Threading.EventWaitHandle> API reference.
+
+### AutoResetEvent class
+
+The <xref:System.Threading.AutoResetEvent?displayProperty=nameWithType> class derives from <xref:System.Threading.EventWaitHandle> and represents a synchronization event that, when signaled, resets automatically to an unsignaled state after releasing a single waiting thread.
+
+The <xref:System.Threading.AutoResetEvent> class cannot represent a named system synchronization event and cannot be used for the inter-process synchronization.
+
+For more information, see the [AutoResetEvent](autoresetevent.md) article and the <xref:System.Threading.AutoResetEvent> API reference.
+
+### ManualResetEvent and ManualResetEventSlim classes
+
+The <xref:System.Threading.ManualResetEvent?displayProperty=nameWithType> class derives from <xref:System.Threading.EventWaitHandle> and represents a synchronization event that, when signaled, stays in a signaled state until its <xref:System.Threading.EventWaitHandle.Reset%2A> method is called.
+
+The <xref:System.Threading.ManualResetEvent> class cannot represent a named system synchronization event and cannot be used for the inter-process synchronization.
+
+The <xref:System.Threading.ManualResetEventSlim?displayProperty=nameWithType> class is a lightweight alternative to <xref:System.Threading.ManualResetEvent>.
+
+For more information, see the [ManualResetEvent and ManualResetEventSlim](manualresetevent-and-manualreseteventslim.md) article and the <xref:System.Threading.ManualResetEvent> or <xref:System.Threading.ManualResetEventSlim> API reference.
+
+### CountdownEvent class
+
+The <xref:System.Threading.CountdownEvent?displayProperty=nameWithType> class represents an event that becomes set when its count is zero.
+
+While <xref:System.Threading.CountdownEvent.CurrentCount?displayProperty=nameWithType> is greater than zero, a thread that calls <xref:System.Threading.CountdownEvent.Wait%2A?displayProperty=nameWithType> is blocked. Call <xref:System.Threading.CountdownEvent.Signal%2A?displayProperty=nameWithType> to decrement an event's count.
+
+In contrast to <xref:System.Threading.ManualResetEvent> or <xref:System.Threading.ManualResetEventSlim>, which you might use to unblock multiple threads with a signal from one thread, you might use <xref:System.Threading.CountdownEvent> to unblock a single thread with signals from multiple threads.
```

Comment:
"single thread" is dubious, there can be multiple threads waiting for the count to reach 0.